html/mycakeapp/webroot/css/seat.css

```
@@ -0,0 +1,72 @@
+.seat_heading {
+    color: #707070;
+    text-align: center;
+    margin-bottom: 20px;
+}
+.seat_heading h1 {
+    font-size: 20px;
+    margin-bottom: 10px;
+}
+.seat_heading h2 {
+    font-size: 18px;
+}
+.seat_heading span {
+    font-size: 14px;
+}
+.seat_select {
+    width: 496px;
+    margin: 0 auto;
+    color: #707070;
+    margin-bottom: 50px;
+}
+.seat_select li {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr;
```

修正不要
```suggestion
    grid-template-columns: repeat(13, 1fr);
```
こちらの方が簡潔にかけます。

---

対応いたしました。